v1-2/learn/how-to-deploy-and-run-ballerina-programs.md

```diff
@@ -613,22 +661,28 @@ $ curl -v -X POST -d '{"name":"Alice", "age":20,"ssn":123456789,"employeeId":1}'
 
 |**Annotation Name**|**Description**|**Default value**|
 |--|--|--|
-|name| Name of the job|\<output file name\>-job|
-|labels| Labels for job|"app: \<outputfilename\>"|
-|restartPolicy| Restart policy|Never|
-|backoffLimit| Restart tries before termination|3|
-|activeDeadlineSeconds| Active deadline seconds|20|
-|schedule| Schedule for cron jobs|none|
+|name|Name of the job|<BALLERINA_FILE_NAME>-job or <BALLERINA_MODULE_NAME>-job|
+|labels|Labels for the job|{ app: <OUTPUT_FILE_NAME> }|
+|annotations|Metadata Annotations map|{}|
+|dockerHost|Docker host IP and docker PORT.(e.g "tcp://192.168.99.100:2376")|DOCKER_HOST environment variable. If DOCKER_HOST is unavailable, uses "unix:///var/run/docker.sock" for Unix or uses "npipe:////./pipe/docker_engine" for Windows 10 or uses "localhost:2375"|
+|dockerCertPath|Docker cert path|DOCKER_CERT_PATH environment variable|
+|registry|Docker registry url|null|
+|username|Username for the docker registry|null|
+|password|Password for the docker registry|null|
+|baseImage|Base image to create the docker image|ballerina/jre8:v1|
+|image|Docker image with tag|<OUTPUT_FILE_NAME>:latest. If field `registry` is set then it will be prepended to the docker image name as <registry>/<OUTPUT_FILE_NAME>:latest|

```

Comment:
```suggestion
|image|Docker image with tag|<OUTPUT_FILE_NAME>:latest. If field `registry` is set, then it will be prepended to the Docker image name as <registry>/<OUTPUT_FILE_NAME>:latest|

```